tslint.json

```diff
@@ -14,7 +14,7 @@
       "static-before-instance",
       "variables-before-functions"
     ],
-    "no-stateless-class": true,
+    "no-unnecessary-class": true,
```

Comment:
This was flagged by `tslint` as an upcoming deprecation:

```
$ tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}
Warning: no-stateless-class rule is deprecated. Replace your usage with the TSLint no-unnecessary-class rule.
...

```

Reply:
I love that new flag name!